scripts/update/01-run.sh

```diff
@@ -27,15 +27,13 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
     echo "============================================="
     echo
 
-    # In Umbrel OS v0.1.2, we need to bind Avahi to only
-    # eth0,wlan0 interfaces to prevent hostname cycling
-    # https://github.com/getumbrel/umbrel-os/issues/76
-    # This patch can be safely removed from Umbrel v0.3.x+
-    if [[ $UMBREL_OS == "v0.1.2" ]] && [[ -f "/etc/avahi/avahi-daemon.conf" ]]; then
-        echo "Binding Avahi to eth0 and wlan0"
-        sed -i "s/#allow-interfaces=eth0/allow-interfaces=eth0,wlan0/g;" "/etc/avahi/avahi-daemon.conf"
-        systemctl restart avahi-daemon.service
-    fi
+    # Older versions of Umbrel OS (<0.3.1) didn't have karen as systemd service
+    # This installs the new karen systemd service and makes sure systemd services are always updated.
+    UMBREL_SYSTEMD_SERVICES=$(ls ${UMBREL_ROOT}/.umbrel-${RELEASE}/scripts/umbrel-os/services/*.service)
+    for service in $UMBREL_SYSTEMD_SERVICES; do
+      install -m 644 "${service}"   "/etc/systemd/system/${service}"
```

Comment:
```suggestion
      install -m 644 "${service}" "/etc/systemd/system/${service}"
```

Reply:
That's the same in Umbrel OS, so that style fix can also be applied there.